src/lib/web-api.ts

```diff
@@ -69,19 +70,48 @@ function createWebCollection<Id, Value>(path: string): CollectionApi<Id, Value>
         },
     });
 }
-export const getWebApi = lazy(async (): Promise<DBApi | undefined> => {
+
+async function createDeployedCollection<Id, Value>(path: string): Promise<CollectionApi<Id, Value>> {
+    const url = new URL(path, SITE_URL).href;
+    const res = await fetch(url);
+    if (res.status !== 200) {
+        throw new Error(res.statusText);
+    }
+    const map = new Map<Id, Value>();
+    const data = (await res.json()) as Record<string, Value>[];
+    for (const [id, value] of Object.entries(data)) {
+        map.set(id as Id, value as Value);
+    }
+    return notifyOnWrite(new SessionStorageMapCollection(path, map), {
```

Comment:
`SessionStorageMapCollection` **ignores** the given map if the cache key is already present. So you're doing all that work for nothing after fetching our data the first time.

Reply:
Changed this so that it will only add stuff that wasn't  found from the api request, meaning if there's new stuff from a new deploy it will get refreshed, but all the new stuff a user adds will still exist